### DISCUSSION TITLE
Fonctionnalité du clasement

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#706] [Nouvelles idées]
Accorder à toutes les personnes ayant commencé leur aventure (réaction quelconque à la lettre du roi) une place dans le classement et ceux sans limite minimale de points afin de se faire une meilleure idée du nombre de joueurs jouant ou ayant joué à DraftBot.

**Comment by Surdidi** [comment]
Je suis entièrement d'accord avec toi, UNE HONTE que ça n'a pas été pensé par le développeur.

**Comment by Pietagorh** [comment]
Ou sinon y'a déjà une issue pour afficher NON CLASSÉ à la place d'un truc au dessus du nombre de joueurs classés

**Reply by Surdidi** [reply]
En fait ce que je trouve particulièrement gênant c'est que l'on ne puisse pas voir le classement général sans être sois même classé.

**Reply by Thesmokingllama** [reply]
En soi c'est pas vraiment gênant si on considère qu'au bout du second report effectué on obtient une place réelle dans le classement, sans compter le fait qu'il n'y a aucun intérêt à pouvoir voir le classement sans avoir un minimum joué à draftbot. La sortie évoquée par Pietagorh me semble parfaite pour résoudre ce problème.

**Reply by BastLast** [reply]
> En soi c'est pas vraiment gênant si on considère qu'au bout du second report effectué on obtient une place réelle dans le classement, sans compter le fait qu'il n'y a aucun intérêt à pouvoir voir le classement sans avoir un minimum joué à draftbot. La sortie évoquée par Pietagorh me semble parfaite pour résoudre ce problème.

perso je pense que ca a un interet pr un joueur de voir le classement avant ca peut le motiver et de manière général c'est pas ouf de se taper une erreur sur une ommande de base quand tu viens juste de commencer

**Reply by BastLast** [reply]
Est-ce que quelqu'un peut créer une issue pour mon com ci dessus ?
